api/all/src/main/java/io/opentelemetry/api/logs/Logger.java

```diff
@@ -33,4 +33,18 @@ public interface Logger {
    * LogRecordBuilder#emit()}.
    */
   LogRecordBuilder logRecordBuilder();
+
+  /**
+   * Return a {@link LogRecordBuilder} to emit an event.
+   *
+   * @param eventName the event name, which identifies the class or type of event. Event with the
+   *     same name are structurally similar to one another. Event names are subject to the same
+   *     naming rules as attribute names. Notably, they are namespaced to avoid collisions. See <a
+   *     href="https://opentelemetry.io/docs/specs/semconv/general/events/">event.name semantic
+   *     conventions</a> for more details.
+   */
+  default EventBuilder eventBuilder(String eventName) {
+    // TODO could implement this on top of logRecordBuilder, or return a no-op implementation
+    throw new UnsupportedOperationException();
```

Comment:
thoughts?

Reply:
The precedent is to have a noop implementation. See example [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/metrics/DoubleHistogramBuilder.java#L47-L49):

```
  default DoubleHistogramBuilder setExplicitBucketBoundariesAdvice(List<Double> bucketBoundaries) {
    return this;
  }
```

We added this method after the initial release, and just don't do anything rather than throw. As a general principle, the API should never throw. 

Besides this, there's the question of where this ought to live until its stable, and the `api-incubator` seems to be the right place. But perhaps you know this and are trying to show how the stable API might evolve to include the new operation.